source/dotnet/Library/AdaptiveCards.Templating/AdaptiveCardsTemplateVisitor.cs

```diff
@@ -575,6 +575,7 @@ public AdaptiveCardsTemplateResult ExpandTemplatedString(ITerminalNode node, boo
             if (HasDataContext())
             {
                 DataContext currentDataContext = GetCurrentDataContext();
+                var tempstring = node.Symbol.InputStream.GetText(new Interval(node.Symbol.StartIndex, node.Symbol.StopIndex));
```

Comment:
is this needed, or was it just for debugging?

Reply:
Thank you, Paul. You're correct. This is for debugging. The branch was created quite a while ago and was used for debugging another issue. Thanks